client/my-sites/activity/filterbar/index.jsx

```diff
@@ -16,14 +14,10 @@ import DateRangeSelector from './date-range-selector';
 import ActionTypeSelector from './action-type-selector';
 import { updateFilter } from 'state/activity-log/actions';
 
-const DATE_FORMAT = 'YYYY-MM-DD';
 export class Filterbar extends Component {
 	state = {
 		showActivityTypes: false,
 		showActivityDates: false,
-		fromDate: null,
-		toDate: null,
-		enteredToDate: null,
 	};
 
 	toggleDateRangeSelector = () => {
```

Comment:
Is there a reason why we need to keep this state in this component? 
Can we also move it into the dateRangeSelector component?

Reply:
The same goes for the ActivityType selector.